tests/test_validate_public_release.py

```
@@ -2336,26 +2339,30 @@ def test_source_stage_validator_commands_are_bound_to_the_snapshot(self) -> None
+            )
+            for plugin in self.module.SOURCE_STAGE_VALIDATED_PLUGINS
+        ] + [
+            (relative, ())
+            for relative in self.module.SOURCE_STAGE_COMMON_VALIDATOR_PATHS
+        ]
+        self.assertEqual(run.call_count, len(expected))
+        for (relative, flags), call in zip(
+            expected,
             run.call_args_list,
-            strict=True,
         ):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)